contracts/staking/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "staking"
+version = "0.1.0"
```

Comment:
Let's use the same version for all the contracts for quick search/replace version bumping

Reply:
Okay, done. And updated `reflect` as well.